test/test_data/http_logging.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "proxy_wasm_intrinsics.h"
+
+class LoggingStreamContext : public Context {
```

Comment:
optional: shorten name to `LoggingContext`. The naming convention in the C++ host and SDKs seems to be `FooContext` for stream contexts, and FooRootContext` for root contexts.

Reply:
Thanks, done. I noticed Rust prefers HttpContext to StreamContext, so another reason to do this.